### DISCUSSION TITLE
Apple M1 machines downtime

### DISCUSSION THREAD

**Discussion by helinanever** [codemagic-ci-cd/codemagic-docs#1482] [Announcements]
We've had an incident whereby the Apple M1 machines were unavailable from 11:20 PM to 7:30 AM UTC. The machines are back up now but it will take some time to resolve the build queue and start all builds. We apologize for the inconvenience caused.

**Comment by suhairhassan** [comment]
Thanks for resolving the issue

**Comment by konnyaku256** [comment]
Thanks! 👏

**Comment by irby** [comment]
Is this issue happening again? Our iOS builds have been stuck in queued mode for the past several minutes.

**Reply by irby** [reply]
Linking to current issue: https://github.com/orgs/codemagic-ci-cd/discussions/2388
